src/applications/check-in/components/pages/Edit/PhoneNumber.jsx

```diff
@@ -133,7 +133,7 @@ export default function PhoneNumber(props) {
         maxlength={null}
         name={`${key}-extension`}
         value={extension}
-        onVaChange={onExtensionChange}
+        onInput={onExtensionInput}
```

Comment:
Here's a screenshot of both of these `VaTextInput` components firing the `input` event:

![image](https://user-images.githubusercontent.com/2008881/163026513-a64db44c-62b2-4950-bbc4-c82f3a0eaafb.png)